x/vesting/types/clawback_vesting_account.go

```diff
@@ -81,7 +81,23 @@ func (va ClawbackVestingAccount) GetVestingCoins(blockTime time.Time) sdk.Coins
 //
 // lockedCoins = totalAmt - unlockedVested
 func (va ClawbackVestingAccount) LockedCoins(blockTime time.Time) sdk.Coins {
-	return va.OriginalVesting.Sub(va.GetUnlockedVestedCoins(blockTime)...)
+	// Can delegate lockedUpVested coins and this will reduce the bank balance
+	// of the account. As long as there're lockedUpVested coins, we'll consider
+	// the delegated tokens as lockedUpVested tokens
+	// min(lockedUpVested, DelegatedFree)
+	//
+	// Consider that the "DelegatedFree" coins tracked on delegations refer to unvested tokens.
+	// These "free" (unvested) tokens can be locked up or unlocked
+	lockedUpVestedDelegatedCoins := va.DelegatedFree.Min(va.GetLockedUpVestedCoins(blockTime))
```

Comment:
This piece is not clear to me. Why are we considering the minimum between delegated free and locked up vested? We are not sure that the second terms are delegated coins